MediaBrowser.Common/Configuration/EncodingConfigurationExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System.IO;
+using MediaBrowser.Model.Configuration;
+
+namespace MediaBrowser.Common.Configuration
+{
+    /// <summary>
+    /// Class containing extension methods for working with the encoding configuration.
+    /// </summary>
+    public static class EncodingConfigurationExtensions
+    {
+        /// <summary>
+        /// Gets the encoding options.
+        /// </summary>
+        /// <param name="configurationManager">The configuration manager.</param>
+        /// <returns>The encoding options.</returns>
+        public static EncodingOptions GetEncodingOptions(this IConfigurationManager configurationManager)
+            => configurationManager.GetConfiguration<EncodingOptions>("encoding");
```

Comment:
Since this configuration only has one option why not remove it? I can fix the web client. We also want to migrate to a single config for the server anyway.

Reply:
It has more than one option.
https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Model/Configuration/EncodingOptions.cs

Reply:
I think this should remain UI configurable, it's a common setting to change.

Reply:
It still is.

Reply:
I know just replying to @dkanada's thoughts :-)

Reply:
I didn't mean remove the option, I just meant we might want to move it to the base configuration. That depends on if we want a single config file or multiple though.

Reply:
This should be a separate PR, IMHO. This one is a bugfix.